crates/rspack_loader_swc/src/compiler.rs

```diff
@@ -374,7 +374,7 @@ impl SwcCompiler {
         let help_msg = formatdoc!{"
           The version of the SWC Wasm plugin you're using might not be compatible with `builtin:swc-loader`.
           The `swc_core` version of the current `rspack_core` is {swc_core_version}. 
-          Please check the `swc_core` version of SWC Wasm plugin to make sure these versions are within the compatible range.
+          Please visit https://plugins.swc.rs/versions/from-core/{swc_core_version} to verify the `swc_core` version of the SWC Wasm plugin and ensure it falls within the compatible range.
```

Comment:
Should strip the prefix `=` if available.

Reply:
Done~